core/amber/src/main/scala/edu/uci/ics/texera/web/WebUtils.scala

```diff
@@ -9,6 +9,16 @@ import edu.uci.ics.amber.clustering.ClusterListener
 
 object WebUtils {
 
+  // TODO change token path
+  def getTokenPath(): String = {
+    ""
+  }
+
+  // TODO change credential path
+  def getCredentialPath(): String = {
+    ""
+  }
+
```

Comment:
hard code to read from a config file, for example `core/conf/web-config.yml`. please check if that file is used in new engine or not.